apps/webstore/src/app/pages/products/product-pages/family-tree/family-tree.component.html

```diff
@@ -30,7 +35,11 @@
         *ngIf="isMutable"
         data-cy="product-options"
     >
-        <div class="blocker" *ngIf="isMobileOptionOpen" (click)="showOptions()"></div>
+        <div
+            class="blocker"
+            *ngIf="isMobileOptionOpen"
+            (click)="showOptions(); eventsService.create('webstore.family-tree.clicked.show-options.mobile-background')"
```

Comment:
Honestly, since we are only checking events on webstore, might it not be a good idea to remove the webstore part to make the call shorter? 

I guess we could have other events in the blogpage or other new applications, but still. Just a consideration to make the calls smaller.

Reply:
@Teodor25 it has crossed my mind. Size-wise, it will be the same or even bigger since we still need to store info about which app it came from. We can have it take less space but we save a few bytes in exchange for a less readable entry